### PR TITLE
64bits

### PR DESCRIPTION
**Pull request by psq**
I implemented support for 64 bits numbers, if you are interested to merge it in, including the rebuilt workflow directly usable with Alfred.

### CODE REVIEW

**Comment by jhunt** [comment]
Thanks for this!  Can you amend the PR to not include the binary workflow?  I'll compile that and verify it after I merge the PR.

**Comment by psq** [comment]
PR no longer includes my rebuilt binary

File: hex.c [hunk 1]
```diff
@@ -232,6 +237,7 @@ int main(int argc, char **argv)
 		}
 	}
 
+
```
Comment (jhunt) [inline hex.c]:
```suggestion
```

Comment (jhunt) [reply hex.c]: This newline is superfluous; once it's removed we can merge this request and get it out to our legions of adoring fans. 😁

**Review by jhunt — CHANGES_REQUESTED** [review]
Love it!  Super minor nit re: newly introduced white space.

File: hex.c (hunk 1 above)
Comment (psq) [reply hex.c]: addressed in e592ce6

**Review by jhunt — APPROVED** [review]
LGTM

**Pull request merged**